mteb/tasks/Abstention/fra/SyntecRerankingAbstention.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskAbstention import AbsTaskAbstention
+from ...Reranking.fra.SyntecReranking import SyntecReranking
+
+
+class SyntecRerankingAbstention(AbsTaskAbstention, SyntecReranking):
+    abstention_task = "Reranking"
+    metadata = TaskMetadata(
+        name="SyntecRerankingAbstention",
+        description="This dataset has been built from the Syntec Collective bargaining agreement.",
+        reference="https://huggingface.co/datasets/lyon-nlp/mteb-fr-reranking-syntec-s2p",
+        dataset={
+            "path": "lyon-nlp/mteb-fr-reranking-syntec-s2p",
+            "revision": "b205c5084a0934ce8af14338bf03feb19499c84d",
+        },
+        type="Abstention",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["fra-Latn"],
+        main_score="map",
```

Comment:
```suggestion
        main_score="map",
```
Unsure if you are proposing map as the main score (given the discussion around AUC)